src/go/k8s/tests/e2e/create-topic-with-client-auth/00-assert.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cluster-tls
+status:
```

Comment:
I don't see in any test that `tlscert` volume mount is verified. Could we add? The best place would be `cluster_controller_test.go`

Reply:
done https://github.com/vectorizedio/redpanda/pull/795/commits/bcd3a1a24bc4015978a90f90b3ca6a27f93525ad